processor/deltatocumulativeprocessor/testdata/tracking/generate.go

```diff
@@ -0,0 +1,101 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+//go:generate go run .
```

Comment:
Do we need to run this script often? Can't we just generate the testdata once and be done with it? 🤔

Reply:
I agree. I would vote that we either do static testing with manually curated files. Or fuzz testing.

Reply:
Generally agreed, that's why I opted to check-in a yaml instead of having it generated during test.

However, the complexity of this yaml is a bit higher than the others, because you need to do the accumulation manually to get the "output" result, making edits really time consuming.

I figured having a generator is a good middle ground here. Happy to be convinced otherwise, but maintaining this large yaml file by hand doesn't sound optimal either

Reply:
> Do we need to run this script often?
I don't expect this to be run often at all, but also figured not publishing it makes the life of whoever needs to edit this harder

Reply:
> However, the complexity of this yaml is a bit higher than the others, because you need to do the accumulation manually to get the "output" result, making edits really time consuming.

Before understanding the testar package I agree the YAML files looked weird, now they look pretty ok to me and if I have to write another testcase I'll just copy-paste an existing folder and do small modifications manually.

I'm not a fan of keeping dead code laying around because it doesn't age well. Eventually nobody understands why it is there and we are all afraid of deleting it thinking "what if I break something if I delete?"

Reply:
removed.